src/test/frontend/db/model_test.cljs

```diff
@@ -82,4 +82,33 @@
 ;;       1 (count a-ref-blocks)
 ;;       (set ["b" "c"]) (set alias-names))))
 
+(deftest ^:focus get-pages-that-mentioned-page-with-show-journal
```

Comment:
Normally we remove metadata after using them since it can effect someone else trying to run the same focused tests. I don't mind removing this one but would be good to remove these in the future